### PR TITLE
Refactoring/code clean up/server

### PR DESCRIPTION
**Pull request by vukkop**

### CODE REVIEW

File: server/helpers/imageHelpers.js
Comment (vukkop) [inline server/helpers/imageHelpers.js:1]: Formatted and rearranged functions

File: server/controllers/image.controler.js
Comment (vukkop) [inline server/controllers/image.controler.js:1]: reformatted

File: client/src/components/admin-components/ListingForm.jsx
Comment (vukkop) [inline client/src/components/admin-components/ListingForm.jsx:1]: Resetting the form on submission

File: server/routes/image.routes.js
Comment (vukkop) [inline server/routes/image.routes.js:1]: deleted empty file as we are not going to use it

File: server/server.js
Comment (vukkop) [inline server/server.js:1]: deleted comments

**Review by BraxtonWalters — APPROVED** [review]
looks good buddy

**Pull request merged**